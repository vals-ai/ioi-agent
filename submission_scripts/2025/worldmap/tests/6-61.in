version https://git-lfs.github.com/spec/v1
oid sha256:9a85ac22696f2df90966d8543933eb3694db8b2a82bbe45c530118a41ee72d08
size 5511
