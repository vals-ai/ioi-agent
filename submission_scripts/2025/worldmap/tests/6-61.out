version https://git-lfs.github.com/spec/v1
oid sha256:3277c2c9c1e625369f0213fcb0426de643c101b4cb4d0d1894d2e110c58276cc
size 181602
